design-document.md

```diff
@@ -106,6 +123,132 @@ to achieve these goals:
    function output itself. {{questions-and-params}} discusses the leakage profiles of 
    various aggregation functions in more detail.
 
+### Threat model
+
+In this section, we enumerate the actors participating in the Prio system and
+enumerate their assets (secrets that are either inherently valuable or which
+confer some capability that enables further attack on the system), the
+capabilities that a malicious or compromised actor has, and potential
+mitigations for attacks enabled by those capabilities.
+
+#### Client
+
+##### Assets
+
+1. Unshared data. Clients are the only actor that can ever see the original
+   data.
+1. Unencrypted data shares.
+1. Client identity (optionally).
+
+##### Capabilities
+
+1. Individual clients can trivially and undetectably defeat the privacy property
+   by leaking data outside of the Prio system. Accordingly, such attacks are
+   outside of the threat model.
+1. Clients may affect the quality of aggregations by reporting false data.
+
+[[OPEN ISSUE: Is there anything that could be done to either mitigate or detect
+a client compromising privacy? Do users have any choice but to trust that
+clients have faithfully implemented the protocol?]]
+
+##### Mitigations
+
+1. The shared validity proof evaluated by aggregators prevents either individual
+   clients or coalitions of clients from compromising the robustness property.
+1. Aggregators can require a minimum number of participating clients in order to
+   minimize the impact of false data.
+1. Many deployments could allow anyone to contribute data shares to aggregators,
+   but others could require that clients establish a trusted identity before
+   they may contribute data. A variety of techniques providing different degrees
+   of assurance could be employed to establish trust in clients, ranging from
+   whitebox cryptography to reproducible builds and remote attestation of
+   software measurements from a trusted computing base.
+
+[[OPEN ISSUE: The problem of clients sending bogus data exists in any metrics
+system, orthogonally to Prio, so perhaps it is out of scope.]]
```

Comment:
I'm not so sure, because at least one mitigation could be applied within the Prio protocol: requiring client identities. If clients had to authenticate to aggregators (or to a trusted authenticator who would then relay anonymized data shares to aggregators), then the system could detect clients sending excessive contributions and reject them.